.github/workflows/visual_tests.yml

```diff
@@ -0,0 +1,87 @@
+name: visual tests
+
+on:
+  push:
+    branches: [main, develop]
+  pull_request:
+    branches: [main, develop]
```

Comment:
do we need it on main? what's the use case for that, hotfix release?

Reply:
Is there a way to filter the commit path so we only run this on commits that impact the templates? (Although I guess that might be hard to determine, since it wouldn't just be templates since view or even model changes could potentially impact the html.) Asking because I saw something about usage limits somewhere.

Reply:
i was wondering the same about `main`! from percy docs it appeared that the standard practice was to leave it on, but set percy to "auto-approve" the build, which I don't really understand. I'm fine with only doing `develop` for now.

re: limiting to paths, we could take the opposite approach and _exclude_ some paths (docs, etc), but generally when touching these files I just put the magic `[ci skip]` in the commit message so that it skips the entire gh actions build.

Reply:
ooh, I wasn't aware of that particular magic incantation

I was thinking more of the case where you want tests to run but don't need visual testing. Probably not something we need right now!  We can always tune it later, especially if the usage becomes a problem.

Reply:
yeah, it's a convention i think from travis that github actions also adopted. the [allowed magic words are described in this post](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/).